File: src/main/java/com/bot/commands/OwnerCommand.java
package com.bot.commands;

import com.bot.utils.CommandCategories;

public abstract class OwnerCommand extends BaseCommand {

    public OwnerCommand() {
        this.ownerCommand = true;
        this.guildOnly = false;
        this.hidden = true;
        this.category = CommandCategories.OWNER;
    }
}

File: src/main/java/com/bot/commands/traditional/OwnerCommand.kt
package com.bot.commands.traditional

import com.bot.utils.CommandCategories

abstract class OwnerCommand : BaseCommand() {
    init {
        this.ownerCommand = true
        this.guildOnly = false
        this.hidden = true
        this.category = CommandCategories.OWNER
        this.canSchedule = false
    }
}
